README.md

```diff
@@ -1 +1,68 @@
-# conda-recipes
+# Conda Recipes
+[![Travis Build Status](https://travis-ci.org/RNAer/conda-recipes.png?branch=master)](https://travis-ci.org/RNAer/conda-recipes)
+
+Conda is a package manager application that quickly installs, runs, and updates packages and their dependencies. It was originally for Python virtual environment management, but it can essentially handle any type of packages in a isolated virtual environment.
+
+This repository hosts conda recipes for various biocore packages or dependencies. The idea is to make bioinformatic tools readily installable and host them on [anaconda cloud](http://anaconda.org).
+
+This will make software installation much easier for users who do not have much command line experience.
+
+Currently, the following packages (and their versions) are built and available on anaconda organization named [biocore](https://anaconda.org/biocore):
+
+* HMMER: [![Binstar Badge](https://anaconda.org/biocore/hmmer/badges/version.svg)](https://anaconda.org/biocore/hmmer)
+
+* Infernal: [![Binstar Badge](https://anaconda.org/biocore/infernal/badges/version.svg)](https://anaconda.org/biocore/infernal)
+
+* Prodigal: [![Binstar Badge](https://anaconda.org/biocore/prodigal/badges/version.svg)](https://anaconda.org/biocore/prodigal)
+
+
+
+To install a package, you just need to run:
+
+    conda install -c https://anaconda.org/biocore <package-name>
+
+No endless headache of time-consuming compilation and missing dependecies any more!
+
+
+## Create a new conda recipe
+To add a new package to the pool, you need to create a recipe to tell conda how to build it.
+
+How to create a recipe is documented on [conda website](http://conda-test.pydata.org/docs/build.html).
+
+You can also borrow from the examples located in this repository: [recipes](https://github.com/biocore/conda-recipes/tree/master/recipes)
```

Comment:
I'd also link to conda's conda-recipes repo (https://github.com/conda/conda-recipes/), which has a lot of examples.